From 962ec8f3bc0a5bc457e61042c2637f0ecfd9c9b6 Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Sat, 30 Jul 2016 12:24:34 +0300
Subject: [PATCH 04/24] ARM64: dts: r8a7796-salvator-x: Fix a typo in the
 hdmi-in node name

The "hdmi-in" I2C address is 0x34, use it the device node name.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 arch/arm64/boot/dts/renesas/r8a7796-salvator-x.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7796-salvator-x.dts b/arch/arm64/boot/dts/renesas/r8a7796-salvator-x.dts
index 7fd4344..965afde 100644
--- a/arch/arm64/boot/dts/renesas/r8a7796-salvator-x.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7796-salvator-x.dts
@@ -604,7 +604,7 @@
 		};
 	};
 
-	hdmi-in@70 {
+	hdmi-in@34 {
 		compatible = "adi,adv7482";
 		reg = <0x34>;
 
-- 
1.9.3

